<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefaultGameOptionsPage</class>
 <widget class="QWidget" name="DefaultGameOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>575</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string notr="true">Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioDisabled">
         <property name="text">
          <string>No Default</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">defaultOptionsMode</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radioNoAutojump">
         <property name="text">
          <string>No Autojump</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">defaultOptionsMode</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="radioFullText">
         <property name="text">
          <string>Full Text</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">defaultOptionsMode</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>radioDisabled</tabstop>
  <tabstop>radioNoAutojump</tabstop>
  <tabstop>radioFullText</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="defaultOptionsMode"/>
 </buttongroups>
</ui>
